sdformat_urdf/CMakeLists.txt

```diff
@@ -69,7 +69,9 @@ target_include_directories(sdformat_urdf
 )
 
 # Add sdformat_urdf_plugin module library
-add_library(sdformat_urdf_plugin MODULE
+# not actually a MODULE as workaround for 
```

Comment:
Looks like there's a little whitespace at the end of this line. Mind removing it?

https://github.com/ros/sdformat_urdf/actions/runs/6782242556/job/18449824992?pr=22#step:3:888

Reply:
Done: https://github.com/ros/sdformat_urdf/pull/22/commits/cf8d7a90c254210cf964db02f771a9c3bc2fee9e .